### PR TITLE
feature/BDD-GWT-IP2-175-side-success

### PR DESCRIPTION
**Pull request by Patrycja99**

### CODE REVIEW

File: RestSharpProject/Features/successside.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿Feature: successside
+# link to Zephyr 
+
+@mytag
```
Comment (Iga221) [inline RestSharpProject/Features/successside.feature:4]: You can delete it.

Comment (Iga221) [reply RestSharpProject/Features/successside.feature:4]: @mytag

File: RestSharpProject/Features/successside.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Scenario: The user makes a successful registration request
```
Comment (Iga221) [inline RestSharpProject/Features/successside.feature:5]: The name of scenario should match the schema.

Comment (kmiecikmarcin) [reply RestSharpProject/Features/successside.feature:5]: I agree with @Iga221 about name of scenario. Example for you: HOME_PAGE_1_IP2-93_home_page_displayed_correctly.

File: RestSharpProject/Features/successside.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿Feature: successside
+# link to Zephyr
```
Comment (kmiecikmarcin) [inline RestSharpProject/Features/successside.feature:2]: not zephyr link in this place. This is space for links about issue's from QA and JS Teams

File: RestSharpProject/Features/successside.feature (hunk 2 above)
Comment (kmiecikmarcin) [reply RestSharpProject/Features/successside.feature:5]: Maybe one unsuccessful scenario?

**Review by kmiecikmarcin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**